cogs/ecosystem.py

```diff
@@ -140,39 +126,37 @@ def faq_nodelist():
-
-- While the EF undeniably plays a big role currently, the ERG ecosystem is becoming more decentralized and independent of the Foundation or any other centralized entity
-     - This decentralization will increase over time as more third-party developers bring value to the Platform.
-     - The Platform is open source and permissionless, allowing anyone to build applications or launch tokens without the permission or assistance of the Foundation.
-     - The EF and has went to great efforts to inform and encourage discussion with the community
-     - If the EF were to vanish, development would not need to stop, and miners could extract and spend all of the Foundation's funds through storage rent.
-
-Therefore, it is difficult to argue that the success of the Platform and the value of ERG tokens necessarily rely on the Foundation or its members. Overall, the value of ERG tokens is becoming more market-driven and independent of the Foundation, which suggests that the fourth criterion of the Howey Test is not met."""
+	df = """
+	Does the Howie Test apply to Ergo?
+	https://ergonaut.space/en/Ergo/Howie
+	"""
 	return(df)
 
 def faq_ergo():
 	df = """Ergo represents the next step in the evolution of proof-of-work blockchain technology, combining Bitcoin’s ledger model with smart contract functionality. Ergo’s extended UTXO design is built to offer secure and robust tooling through the use of its native language, ErgoScript, and its inherent Sigma protocols that utilize non-interactive zero-knowledge proofs.
 
 Ergo’s Autolykos v2 mining consensus algorithm is GPU friendly and ASIC resistant, allowing for anyone with consumer-grade GPU hardware to easily and effectively participate in securing the network.
 
-The Ergo blockchain launched in 2019 with no pre-mining or pre-allocation of any coins. The platform strives to be fair and accessible to all with an emphasis on building truly decentralized financial products and services. The ecosystem currently features the SigUSD stablecoin, ErgoMixer, a cross-chain DEX, and much more. More information can be found at ergoplatform.org and sigmaverse.io."""
+The Ergo blockchain launched in 2019 with no pre-mining or pre-allocation of any coins. The platform strives to be fair and accessible to all with an emphasis on building truly decentralized financial products and services. The ecosystem currently features the SigUSD stablecoin, ErgoMixer, a cross-chain DEX, and much more. More information can be found at ergoplatform.org, sigmaverse.io and ergonaut.space."""
+	return(df)
+
+def faq_newdev():
+	df = """
+	If you are a developer new to the Ergo ecosystem, these links may be useful:
+	- Documentation: https://docs.ergoplatform.com/dev/start/
+	- Developer Channel: https://discord.gg/ergo-platform-668903786361651200
```

Comment:
@cafebedouin this discord link might silence the bot based on ergo server config

Reply:
Seems ok. Looks like the self-assign roles under Rules and Start Here aren't set-up right. So, I'm going to learn it until Glasgow has a chance to resolve.